### PR TITLE
Warning message

### PR DESCRIPTION
**Pull request by tt-le**
Added warning message thrown by test case and explaining that it is expected.

### CODE REVIEW

**Review by AustinSeto — APPROVED** [review]
LGTM

File: README.adoc [hunk 1]
```diff
@@ -319,6 +322,8 @@ Results :
 Tests run: 5, Failures: 0, Errors: 0, Skipped: 0
 ----
 
+The warning message is expected and result from a request to a bad or an unknown host name.
```
Comment (gkwan-ibm) [inline README.adoc:325]: @tt-le better mention the test `testUnknownHost()` causes the warning/error

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
@tt-le better mention the test `testUnknownHost()` causes the warning/error

**Comment by griffinhadfield** [comment]
Verified in lgdev. Looks good to me.

**Review by griffinhadfield — APPROVED** [review]

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**